### DISCUSSION TITLE
Security of Solar Panel Systems

### DISCUSSION THREAD

**Discussion by johanmeijer** [johanmeijer/grott#581] [General]
Herewith a link to a (Dutch, but google will be able to translate I suppose ) article which describes that a Dutch hacker was able to take over 4 miljoen Solar Panel Systems in 150 Countries. 

     https://archive.is/3hJwz 

It is not the story about Growatt but .... be aware that your Solar Panel System can be managed from the Growatt  Servers and the Protocol is not that difficult to Hack. 

Using Grott with the blockcmd = True option (blocking commands from the outside) or using grottserver (without any connections with an outside server) can protect you. 

Of course there are other very valid solutions also (like direct connection with the modbus interface) to project you. I think that having some experience with home servers (running on RPI, NAS systems or other PC's) Grott can be an easy solution without changing a lot within your configuration.

**Comment by tabascoz** [comment]
Agree Johan, thanks for sharing it

**Comment by westerguard** [comment]
I've read that too. Got me thinking again about using Grott server, and making it only locally available.
Thanks for sharing.

**Comment by PedroKTFC** [comment]
I've been using (a personally slightly modified version of) grottserver for months. Growatt has not seen my system in all that time and I've been extremely happy with the situation!
